drivers/bridge/bridge.go

```diff
@@ -98,6 +98,7 @@ type bridgeNetwork struct {
 
 type driver struct {
 	config      *configuration
+	configured  bool
```

Comment:
ok fair enough. lets try to narrow down the focus on fixing the invalid panic in this PR and open a separate thread on dynamic config change.